drops/product.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: default
+title: Product Drop
+has_children: false
+parent: Engine Drops
+permalink: /drops/product
+---
+
+{% include page_header.html %}
+
+### Drop Attributes
+
+- id
+- brand
+- name
+- slug
+- category
+- description
+- price
+- sku
+- meta_title
+- display_image_url
+- option_menu_data
+- variants_by_option_value
+- is_sellable?
+- product_seo_title
+- taxons
+- gift_card?
+- gift_card
+
+### Drop Methods
+
+The Product Drop allows you to access information regarding a specific product
+from your database. You'd generally use this drop in a product partial.
+
+#### display_image_alt
+
+Returns a String containing an image's alternative text.
+
+Using this Drop method is dentical to accessing this information with
+`product.display_image.alt`.
+
+```liquid
+{% raw %}{{ product.display_image_alt }}
+#=> "E-11 Blaster Rifle in action during the Battle of Endor"{% endraw %}
+```
+
+#### display_image_large_url
+
+Returns a String containing the URL for a product's image at 600x600.
+
+```liquid
+{% raw %}{{ product.display_image_large_url }}
+#=> "/path/to/lochnessmonster.png?1234"{% endraw %}
+```
+
+You might use this Drop method to set a background image with inline CSS.
+
+```liquid
+{% raw %}<div class="product-header"
+     style="background-image: url({{ product.display_image_large_url }});">
+  ...
+</div>{% endraw %}
+```
+
+#### display_image_original_url
+
+Returns a String containing the URL for a product's image at its original
+resolution.
+
+```liquid
+{% raw %}{{ product.display_image_original_url }}
+#=> "/path/to/lochnessmonster.png?1234"{% endraw %}
+```
+
+You might use this Drop method to set a background image with inline CSS.
+
+```liquid
+{% raw %}<div class="product-header"
+     style="background-image: url({{ product.display_image_original_url }});">
+  ...
+</div>{% endraw %}
+```
+
+#### display_image_small_url
+
+Returns a String containing the URL for a product's image at 100x100.
+
+```liquid
+{% raw %}{{ product.display_image_small_url }}
+#=> "/path/to/lochnessmonster.png?1234"{% endraw %}
+```
+
+You might use this Drop method to set a background image with inline CSS.
+
+```liquid
+{% raw %}<div class="product-preview"
+     style="background-image: url({{ product.display_image_small_url }});">
+  ...
+</div>{% endraw %}
+```
+
+#### has_embroidery?
+
+Returns a Boolean value of `true` or `false` denoting if a product has an
```

Comment:
Stylistic suggestion: to make the docs more consistent, it would probably be best to use the same wording for all of the methods that return a Boolean, e.g. `Returns a Boolean value of true or false denoting if a product has an
embroidery option.` or `Returns a Boolean denoting if a Payment source is valid.`. I noticed that all of the methods but this one were documented like the latter example, although I think that they are both well written and that you could go with either.

Reply:
In my opinion, "Returns a Boolean value of `true` or `false`" is actually redundant. Good catch!